packages/sdk/src/sdk/services/Auth/UserAuth.ts

```diff
@@ -127,6 +127,16 @@ export class UserAuth implements AuthService {
     return this.hedgehog.isLoggedIn()
   }
 
+  resetPassword = async ({
```

Comment:
I would remove this stuff. @rickyrombo has a PR that's about to change up all the auth classes in SDK and we don't need this change to accomplish what you're doing.